### PR TITLE
Add/Update AWS Audit Manager > CIS Controls v8 IG1. Closes #695

### LINKED ISSUE #695: Add/Update AWS Audit Manager > CIS Controls v8 IG1
**Controls to be removed**

Note: The controls & the entire benchmark, as mentioned below, are removed 

- [x] 13 Network Monitoring and Defense
- [x] 16 Application Software Security

Benchmark | Control(s)
-- | --
13_1 | apigateway_stage_use_waf_web_acl
16_1 | codebuild_project_artifact_encryption_enabled
16_12 | codebuild_project_artifact_encryption_enabled
16_1 | codebuild_project_environment_privileged_mode_disabled
16_12 | codebuild_project_environment_privileged_mode_disabled
16_1 | codebuild_project_logging_enabled
16_12 | codebuild_project_logging_enabled
16_1 | codebuild_project_plaintext_env_variables_no_sensitive_aws_values
16_12 | codebuild_project_plaintext_env_variables_no_sensitive_aws_values
16_1 | codebuild_project_source_repo_oauth_configured
16_12 | codebuild_project_source_repo_oauth_configured
11_3 | ebs_volume_encryption_at_rest_enabled
4_6 | ebs_volume_encryption_at_rest_enabled
13_1 | guardduty_enabled
13_3 | guardduty_enabled
4_6 | iam_account_password_policy_min_length_14
5_2 | iam_account_password_policy_min_length_14
3_3 | s3_bucket_policy_restricts_cross_account_permission_changes
13_1 | securityhub_enabled
13_6 | vpc_flow_logs_enabled
13_6 | wafv2_web_acl_logging_enabled

**Controls to added**
Benchmark | Control(s)
-- | --
11_3 | encrypted_volumes
2_2 | elastic_beanstalk_managed_updates_enabled
3_3 | s3_bucket_policy_grantee_check
3_3 | ssm_document_not_public
4_6 | encrypted_volumes
4_6 | iam_password_policy
5_2 | iam_password_policy
7_3 | elastic_beanstalk_managed_updates_enabled
8_2 | waf_classic_logging_enabled

### PR DESCRIPTION
**Pull request by Priyanka-Chatterjee-2000**

### CODE REVIEW

**Review by rajlearner17 — APPROVED** [review]
LGTM

File: conformance_pack/wafv2.sp [hunk 1]
```diff
@@ -10,7 +10,6 @@ control "wafv2_web_acl_logging_enabled" {
   query       = query.wafv2_web_acl_logging_enabled
 
   tags = merge(local.conformance_pack_wafv2_common_tags, {
-    cis_controls_v8_ig1                    = "true"
```
Comment (madhushreeray30) [inline conformance_pack/wafv2.sp:13]: We should keep this tag. This control is being used in `cis_controls_v8_ig1_8_2`

File: conformance_pack/s3.sp [hunk 2]
```diff
@@ -289,7 +289,6 @@ control "s3_bucket_policy_restricts_cross_account_permission_changes" {
   query       = query.s3_bucket_policy_restricts_cross_account_permission_changes
 
   tags = merge(local.conformance_pack_s3_common_tags, {
-    cis_controls_v8_ig1 = "true"
     nist_csf            = "true"
```
Comment (madhushreeray30) [inline conformance_pack/s3.sp:292]:
```suggestion
    nist_csf = "true"
```

File: conformance_pack/codebuild.sp [hunk 3]
```diff
@@ -69,7 +67,6 @@ control "codebuild_project_environment_privileged_mode_disabled" {
   description = "This control checks if an AWS CodeBuild project environment has privileged mode enabled. This control fails when an AWS CodeBuild project environment has privileged mode enabled."
   query       = query.codebuild_project_environment_privileged_mode_disabled
   tags = merge(local.conformance_pack_codebuild_common_tags, {
-    cis_controls_v8_ig1 = "true"
     nist_csf            = "true"
```
Comment (madhushreeray30) [inline conformance_pack/codebuild.sp:70]:
```suggestion
    nist_csf     = "true"
```

File: conformance_pack/codebuild.sp [hunk 4]
```diff
@@ -69,7 +67,6 @@ control "codebuild_project_environment_privileged_mode_disabled" {
...
     pci_dss_v321        = "true"
```
Comment (madhushreeray30) [inline conformance_pack/codebuild.sp:71]:
```suggestion
    pci_dss_v321 = "true"
```

**Review by madhushreeray30 — COMMENTED** [review]
@Priyanka-Chatterjee-2000 please have a look at the comments, thanks!

**Review by madhushreeray30 — APPROVED** [review]
LGTM 👍

**Review by misraved — APPROVED** [review]

**Pull request merged**